.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   run-unit-tests:
     name: Run Tests
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
We're lucky we're open source, because the MacOS multiplier for actions minutes is crazy 😬

Outside of this, I think it is good practice to only **not** use Linux if really required.